tests/unit/fixtures.py

```diff
@@ -4,7 +4,7 @@
 from unittest.mock import patch
 
 import pytest
-import scenario
```

Comment:
Should we remove scenario from requirements file also?

Reply:
No, `ops-scenario` is an optional extra that needs to be installed on its own.

Reference:
- https://github.com/canonical/operator/blob/main/ops/testing.py#L81